<h1>Содержание<span class="tocSkip"></span></h1>
<div class="toc"><ul class="toc-item"><li><span><a href="#Средняя-величина" data-toc-modified-id="Средняя-величина-1"><span class="toc-item-num">1&nbsp;&nbsp;</span>Средняя величина</a></span><ul class="toc-item"><li><span><a href="#Формула" data-toc-modified-id="Формула-1.1"><span class="toc-item-num">1.1&nbsp;&nbsp;</span>Формула</a></span></li><li><span><a href="#Определение" data-toc-modified-id="Определение-1.2"><span class="toc-item-num">1.2&nbsp;&nbsp;</span>Определение</a></span></li><li><span><a href="#Пример" data-toc-modified-id="Пример-1.3"><span class="toc-item-num">1.3&nbsp;&nbsp;</span>Пример</a></span></li><li><span><a href="#Вывод" data-toc-modified-id="Вывод-1.4"><span class="toc-item-num">1.4&nbsp;&nbsp;</span>Вывод</a></span></li></ul></li><li><span><a href="#Среднее-геометрическое" data-toc-modified-id="Среднее-геометрическое-2"><span class="toc-item-num">2&nbsp;&nbsp;</span>Среднее геометрическое</a></span></li></ul></div>

In [1]:
import pandas as pd

# Виды среднего 

## Средняя величина

### Формула

$$ \overline{x} = \frac{ \Sigma{x_i} }{ n } $$

$ \overline{x} $ - средняя величина

$ \Sigma{x_i} $ - сумма всех значений следуемого признака

$ n $ - количество измерений

### Определение

**Средней величиной** называют наиболее типичное значение исследуемого признака 

### Пример

Допустим, мы хотим найти наиболее типичный рост баскетболиста в команде из 5 человек.

В данном случаи, исследуемый признак — это рост, а количество измерений равно 5 (так — как мы замеряем рост у 5 человек)

In [2]:
data =[['Иван', 1.98],['Илья', 2.04],['Павел', 2.10], ['Олег', 1.97], ['Семён', 2.15]]

В таблице приведены результаты измерений

In [3]:
table = pd.DataFrame(data=data, columns=['Имя игрока','Рост'])

In [4]:
table

Имя игрока  Рост
0       Иван  1.98
1       Илья  2.04
2      Павел  2.10
3       Олег  1.97
4      Семён  2.15

Произведем расчеты 

$$ \overline{x} = \frac{ 1.98 + 2.04 + 2.10 + 1.97 + 2.15 }{ 5 } = \frac{ 10,24 }{ 5 } = 2,048 $$

Также для вычисление среднего можно воспользоваться специальной функцией из pandas, которая называется mean 

In [5]:
table['Рост'].mean()

2.048

### Вывод

Средний значение составляет 2.048. Это значит, что если выбрать любого игрока из команды  и измерить его рост, то результат с высокой вероятностью будет приблизительно равен 2.048.

## Среднее геометрическое